facts/secrets/mt7622-openwrt-example.yaml

```diff
@@ -0,0 +1,21 @@
+# scale
+root_hash: "$1$zh0PjBbB$f9aFGDX9vNYNdSRexhib8/"
+
+# Bump for each year for scale
+scale: 19
```

Comment:
21?

Reply:
Ya let me bump that to `21`, good catch!

Reply:
Updated